src/models/slices/status.ts

```diff
@@ -0,0 +1,3 @@
+export interface StatusState {
```

Comment:
I think we should try and come up with a more descriptive name. As a consumer of the library, I would wonder "status of what"?

Reply:
maybe `SearchStatusState`?

Reply:
Sure, that sounds good

Reply:
updated!